twine/repository.py

```diff
@@ -93,8 +93,10 @@ def _convert_data_to_list_of_tuples(data):
                     data_to_send.append((key, item))
         return data_to_send
 
-    def set_certificate_authority(self, cacert):
-        if cacert:
+    def set_certificate_authority(self, cacert, trusted):
```

Comment:
This is intended to be a public API which means adding a non-optional parameter is backwards incompatible. Please add a default for `trusted`

Reply:
You're right, sorry for this